src/Symfony/Component/Config/Definition/ConfigurationInterface.php

```diff
@@ -22,6 +22,8 @@ interface ConfigurationInterface
 {
     /**
      * Generates the configuration tree builder.
+     *
+     * @return TreeBuilder
      */
-    public function getConfigTreeBuilder(): TreeBuilder;
+    public function getConfigTreeBuilder();
```

Comment:
this one has already been delayed from 6.0 to 7.0, throwing a deprecation warning since 5.3.

Same for the Command execute method. If 2.5 years is not enough to get them to release a version with a return type, what will happen in 8.0 in 2 years ? Will we delay it again ?

Reply:
Indeed. We need to be sure maintainers have received the message. /cc @Seldaek for monolog-bundle and @derrabus I guess for Doctrine?

Good for twig-extra-bundle (but it still is missing a return type for TwigExtraExtension::load)

Reply:
Well, for the void type in load, I'm fine with delaying it to give more time if needed, as that's much newer (we started reporting a deprecation for it in 6.2 only)

Reply:
monolog-bunde updated https://github.com/symfony/monolog-bundle/pull/461